pyro/distributions/transforms/spline.py

```diff
@@ -0,0 +1,352 @@
+# This implementation is adapted in part from:
+# * https://github.com/tonyduan/normalizing-flows/blob/master/nf/flows.py; and,
+# * https://github.com/hmdolatabadi/LRS_NF/blob/master/nde/transforms/nonlinearities.py,
+# under the MIT license.
```

Comment:
Can you add the following license header above your comment:
```
# Copyright Contributors to the Pyro project.
# Copyright (c) 2020 Hadi M. Dolatabadi, Sarah Erfani, Christopher Leckie
# Copyright (c) 2019 Conor Durkan, Artur Bekasov, Iain Murray, George Papamakarios
# Copyright (c) 2019 Tony Duan
# SPDX-License-Identifier: MIT
```

Reply:
Done